src/navigators/RootStack.tsx

```diff
@@ -49,28 +49,31 @@ export function RootStack() {
     const Stack = createNativeStackNavigator<LoggedInRootParamList>();
     const hasAccountAndProject = !!(activeProject?.id && account?.id);
 
-    const initialRoute = !hasAccountAndProject
-      ? 'InviteRequired'
-      : shouldRenderConsentScreen
-      ? 'screens/ConsentScreen'
-      : 'app';
-
-    if (loadingAccountOrProject && initialRoute === 'InviteRequired') {
+    if (loadingAccountOrProject) {
       return (
         <ActivityIndicatorView
-          message={t('root-stack-waiting-for-account-and-project', 'Loading')}
+          message={t(
+            'root-stack-waiting-for-account-and-project',
+            'Loading account',
+          )}
         />
       );
     }
 
-    if (loadingConsents && initialRoute === 'screens/ConsentScreen') {
```

Comment:
I think the reason I did this is because the consents API call never becomes enabled if there is not a valid activeProject which leads to being stuck at this loading indicator indefinitely when we want to push the user to the Invite required screen. Obviously there is something else wrong here though if this resolves the problem you are facing.

Reply:
Yep- I hit that right after I put the PR up and was handling in a separate PR, but have now just committed the rest into this PR.